.github/CODEOWNERS

```diff
@@ -44,121 +47,121 @@ Controls/web.tsx @Vitalius1 @natalieethell
+common/_semanticSlots.scss @phkuo
 common/_themeOverrides.scss @phkuo
 common/_common.scss @phkuo
 
 ### Utilities
-dateMath/     @lorejoh
-dateValues/   @lorejoh
+dateMath/ @lorejoh
+dateValues/ @lorejoh
 
 ## Components
-Accordion/ @JasonGore
-ActivityItem/ @jdhuntington
+Accordion/ @khmakoto
+ActivityItem/ @xugao
 Announced/ @natalieethell
-Breadcrumb/ @jdhuntington
+Breadcrumb/ @xugao
 Button/ @khmakoto
 Calendar/ @lorejoh
 Callout/ @joschect
-react-cards/Card/  @khmakoto
+react-cards/Card/ @khmakoto
 Check/ @KevinTCoughlin
-Checkbox/ @cliffkoh
-ChoiceGroup/ @srideshpande
+Checkbox/ @khmakoto @xugao
+ChoiceGroup/ @ecraig12345
 Coachmark/ @leddie24
-CollapsibleSection/ @JasonGore
+CollapsibleSection/ @khmakoto
 ColorPicker/ @ecraig12345
 ComboBox/ @joschect
 CommandBar/ @micahgodbolt @joschect
 ContextualMenu/ @joschect
 DatePicker/ @lorejoh
 DetailsList/ @KevinTCoughlin
-Dialog/ @kkjeer
+Dialog/ @joschect
 DocumentCard/ @yiminwu
 Dropdown/ @joschect
 Fabric/ @dzearing
-Facepile/ @markionium
+Facepile/ @markionium @mtennoe
 FolderCover/ @ThomasMichon @bigbadcapers
-FocusTrapZone/ @JasonGore
-FocusZone/ @JasonGore
-GroupedList/  @aditima @KevinTCoughlin
-HoverCard/   @atneik @Jahnp @Vitalius1
-Icon/ @dzearing
+FocusTrapZone/ @khmakoto
+FocusZone/ @khmakoto
+GroupedList/ @aditima @KevinTCoughlin
+HoverCard/ @atneik @Jahnp @Vitalius1
+Icon/ @dzearing @ecraig12345
 Image/ @dzearing
-Label/ @cliffkoh
-Layer/  @ThomasMichon @JasonGore
+Label/ @khmakoto
+Layer/ @ThomasMichon
 Link/ @khmakoto
 List/ @KevinTCoughlin
 MarqueeSelection/ @dzearing
 MessageBar/ @ecraig12345
-Modal/ @JasonGore
-office-ui-fabric-react/src/components/Nav/ @jdhuntington
+Modal/ @joschect
+office-ui-fabric-react/src/components/Nav/ @ecraig12345
 OverflowSet/ @micahgodbolt
-Overlay/ @kkjeer
+Overlay/ @aneeshack4
 Pagination/ @caradong
-Panel/ @kkjeer
-Persona/ @markionium
-PersonaCoin/ @mtennoe
-Persona/PersonaConsts.tsx @mtennoe
-pickers/  @joschect
-Pivot/ @jdhuntington
-Popup/ @JasonGore
+Panel/ @aneeshack4
+Persona/ @markionium @mtennoe
+PersonaCoin/ @markionium @mtennoe
+Persona/PersonaConsts.tsx @markionium @mtennoe
+pickers/ @joschect
+Pivot/ @behowell
+Popup/ @joschect
 ProgressIndicator/ @aneeshack4
 Rating/ @jdhuntington
 ResizeGroup/ @micahgodbolt
 SearchBox/ @ecraig12345
 Separator/ @natalieethell
 Shimmer/ @Vitalius1 @atneik
 Signals/ @ThomasMichon
-Slider/ @aneeshack4
-SpinButton/ @aneeshack4
+Slider/ @joschect
+SpinButton/ @khmakoto
 Spinner/ @aneeshack4
-Stack/  @khmakoto @kkjeer
+Stack/ @khmakoto
 SwatchColorPicker/ @ecraig12345
 TeachingBubble/ @micahgodbolt
 Text/ @natalieethell
 TextField/ @ecraig12345
-Theme/ @JasonGore
+Theme/ @dzearing
 Tile/ @ThomasMichon
 TileList/ @ThomasMichon
-office-ui-fabric-react/src/components/Toggle/  @phkuo
-experiments/src/components/Toggle/  @khmakoto @JasonGore
+office-ui-fabric-react/src/components/Toggle/ @phkuo @xugao
+experiments/src/components/Toggle/ @khmakoto
 Tooltip/ @micahgodbolt
 
 ## Docs
 # ActivityItem/docs/
```

Comment:
A little out of scope for this PR but we should probably just remove this section...we never have any separate owners for docs and it's just another thing to get out of sync.